services/blockchain-indexer/app.js

```diff
@@ -46,32 +46,30 @@ const app = Microservice({
 
 setAppContext(app);
 
-(async () => {
-	// Add routes, events & jobs
-	if (config.operations.IndexingMode) {
-		await app.addMethods(path.join(__dirname, 'methods', 'indexer'));
-		await app.addEvents(path.join(__dirname, 'events'));
-	}
+// Add routes, events & jobs
+if (config.operations.IndexingMode) {
```

Comment:
Please update the config property.
```suggestion
if (config.operations.isIndexingModeEnabled) {
```